substrate/client/storage-monitor/src/lib.rs

```diff
@@ -70,10 +69,10 @@ impl StorageMonitorService {
 	/// Creates new StorageMonitorService for given client config
 	pub fn try_spawn(
 		parameters: StorageMonitorParams,
-		database: DatabaseSource,
+		path: Option<&Path>,
```

Comment:
`Option` is not needed here, could just be `Path`.

Reply:
Yes, but then if/else needs to be checked at call site + there is logging that would need to be copied as well. I looked at that and decided to keep it an option.

Reply:
Hm.. `None` may not always mean/require logging, it could be handled in other way. And IMO handling `None` seems like part of caller responsibility.

Reply:
Sure, done in last commit